src/hotspot/share/opto/graphKit.cpp

```diff
@@ -735,6 +735,34 @@ SafePointNode* GraphKit::clone_map() {
   return clonemap;
 }
 
+//-----------------------------destruct_map_clone------------------------------
+//
+// Order of destruct is important to increase the likelyhood that memory can be re-used. We need
+// to destruct/free/delete in the exact opposite order as clone_map().
+void GraphKit::destruct_map_clone(SafePointNode* m) {
+  if (m == nullptr) return;
```

Comment:
Please rename `m` to `sfp`. `m` is for `memory`.

Reply:
Done. Was just following clone_map.